{"uuid":"bc60c982-1a9a-4b04-8ddf-f5b2ccfd2020","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["4eaf5d28-b153-4880-a2a9-860095574beb"],"befores":[],"afters":[{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696389896411,"stop":1696389896411},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696389896546,"stop":1696389896546},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696389896601,"stop":1696389896601},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696389898691,"stop":1696389898691}],"start":1696389896236,"stop":1696389898692}